js/tiles.js

```diff
@@ -99,11 +99,24 @@ PlacedTile.prototype.createImage = function(file) {
 PlacedTile.prototype.playerEnters = function(player) {
   this.images.playerEnters(player);
   this.whenPlayerEnters(player);
+  this.scrollToCenter();
 };
 PlacedTile.prototype.playerLeaves = function(player) {
   this.images.playerLeaves(player);
   this.whenPlayerLeaves(player);
 };
+PlacedTile.prototype.scrollToCenter = function() {
+  const $element = $(this.images.container).children();
+  const offset = $element.offset();
+  // calculate the top and left offsets in order for the tile to be centered
+  const centerTop = window.innerHeight / 2 - 128.157;
```

Comment:
I think, there are variables for these magic numbers. They should be used.

Reply:
Found the variables. I wasn't sure if they were going to work since they were declared in levels.js which is loaded after tiles.js, but it appears that they do.